CARLA-UE5 Window Building error (sqlite3)

This is a persistent issue even after a reinstallation. This occurs during Setup.bat attempts.
(Window 11)


```
Buiding CARLA...
[0/2] Re-checking globbed directories...
[32/414] Building C object CMakeFiles\sqlite3.dir\_deps\sqlite3-src\shell.c.obj
FAILED: CMakeFiles/sqlite3.dir/_deps/sqlite3-src/shell.c.obj
C:\PROGRA~1\MIB055~1\2022\COMMUN~1\VC\Tools\MSVC\1440~1.338\bin\Hostx64\x64\cl.exe  /nologo -D_CRT_SECURE_NO_WARNINGS -IC:\Users\USER\CarlaUE5\Build\_deps\zlib-src -IC:\Users\USER\CarlaUE5\Build\_deps\zlib-build -IC:\Users\USER\CarlaUE5\Build\_deps\libpng-src -IC:\Users\USER\CarlaUE5\Build\_deps\libpng-build /DWIN32 /D_WINDOWS /O2 /Ob2 /DNDEBUG -std:c11 -MD /wd4005 /showIncludes /FoCMakeFiles\sqlite3.dir\_deps\sqlite3-src\shell.c.obj /FdCMakeFiles\sqlite3.dir\ /FS -c C:\Users\USER\CarlaUE5\Build\_deps\sqlite3-src\shell.c
C:\Users\USER\CarlaUE5\Build\_deps\sqlite3-src\shell.c(1): warning C4819: The file contains a character that cannot be represented in the current code page (949). Save the file in Unicode format to prevent data loss
C:\Users\USER\CarlaUE5\Build\_deps\sqlite3-src\shell.c(26303): warning C4819: The file contains a character that cannot be represented in the current code page (949). Save the file in Unicode format to prevent data loss
C:\Users\USER\CarlaUE5\Build\_deps\sqlite3-src\shell.c(26338): error C2001: newline in constant
C:\Users\USER\CarlaUE5\Build\_deps\sqlite3-src\shell.c(26339): error C2143: syntax error: missing ';' before 'const'
C:\Users\USER\CarlaUE5\Build\_deps\sqlite3-src\shell.c(26353): error C2065: 'zBom': undeclared identifier
C:\Users\USER\CarlaUE5\Build\_deps\sqlite3-src\shell.c(26353): warning C4047: '=': 'int' differs in levels of indirection from 'const char *'
C:\Users\USER\CarlaUE5\Build\_deps\sqlite3-src\shell.c(26420): error C2065: 'zBom': undeclared identifier
C:\Users\USER\CarlaUE5\Build\_deps\sqlite3-src\shell.c(26420): error C2065: 'zBom': undeclared identifier
C:\Users\USER\CarlaUE5\Build\_deps\sqlite3-src\shell.c(26420): warning C4047: 'function': 'const char *' differs in levels of indirection from 'int'
C:\Users\USER\CarlaUE5\Build\_deps\sqlite3-src\shell.c(26420): warning C4024: 'oPutsUtf8': different types for formal and actual parameter 1
C:\Users\USER\CarlaUE5\Build\_deps\sqlite3-src\shell.c(26433): error C2065: 'zBom': undeclared identifier
C:\Users\USER\CarlaUE5\Build\_deps\sqlite3-src\shell.c(26433): error C2065: 'zBom': undeclared identifier
C:\Users\USER\CarlaUE5\Build\_deps\sqlite3-src\shell.c(26433): warning C4047: 'function': 'const char *' differs in levels of indirection from 'int'
C:\Users\USER\CarlaUE5\Build\_deps\sqlite3-src\shell.c(26433): warning C4024: 'oPutsUtf8': different types for formal and actual parameter 1
[49/414] Building C object CMakeFiles\libsqlite3.dir\_deps\sqlite3-src\sqlite3.c.obj
ninja: build stopped: subcommand failed.

```

I followed the error code and looked at the lines in the code, but there didn't seem to be any problems.

C:\Users\USER\CarlaUE5\Build\_deps\sqlite3-src\shell.c
![image](https://github.com/carla-simulator/carla/assets/70637892/90390f37-7566-41f1-8883-96caaad59a4f)
![image](https://github.com/carla-simulator/carla/assets/70637892/334458f4-35d1-4586-97f3-3e8780a55ae9)
![image](https://github.com/carla-simulator/carla/assets/70637892/24883a40-5ab5-4d2d-9f7a-3fe6f8493e9e)
![image](https://github.com/carla-simulator/carla/assets/70637892/e5103127-7051-4e3e-ae40-1da962d7c1f2)

hi, I met the same problem. However, when I had changed the way of coding to "UTF-8(BOM",  another problem happened.

[0/2] Re-checking globbed directories...
[1/168] Linking CXX executable _deps\recastnavigation-build\RecastBuilder\RecastBuilder.exe
FAILED: _deps/recastnavigation-build/RecastBuilder/RecastBuilder.exe
C:\Windows\system32\cmd.exe /C "cd . && "C:\Program Files\CMake\bin\cmake.exe" -E vs_link_exe --intdir=_deps\recastnavigation-build\RecastBuilder\CMakeFiles\RecastBuilder.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\mt.exe --manifests  -- C:\PROGRA~1\MIB055~1\2022\COMMUN~1\VC\Tools\MSVC\1440~1.338\bin\Hostx64\x64\link.exe /nologo _deps\recastnavigation-build\RecastBuilder\CMakeFiles\RecastBuilder.dir\Contrib\fastlz\fastlz.c.obj _deps\recastnavigation-build\RecastBuilder\CMakeFiles\RecastBuilder.dir\Source\ChunkyTriMesh.cpp.obj _deps\recastnavigation-build\RecastBuilder\CMakeFiles\RecastBuilder.dir\Source\InputGeom.cpp.obj _deps\recastnavigation-build\RecastBuilder\CMakeFiles\RecastBuilder.dir\Source\MeshLoaderObj.cpp.obj _deps\recastnavigation-build\RecastBuilder\CMakeFiles\RecastBuilder.dir\Source\PerfTimer.cpp.obj _deps\recastnavigation-build\RecastBuilder\CMakeFiles\RecastBuilder.dir\Source\Sample.cpp.obj _deps\recastnavigation-build\RecastBuilder\CMakeFiles\RecastBuilder.dir\Source\SampleInterfaces.cpp.obj _deps\recastnavigation-build\RecastBuilder\CMakeFiles\RecastBuilder.dir\Source\Sample_SoloMesh.cpp.obj _deps\recastnavigation-build\RecastBuilder\CMakeFiles\RecastBuilder.dir\Source\Sample_TileMesh.cpp.obj _deps\recastnavigation-build\RecastBuilder\CMakeFiles\RecastBuilder.dir\Source\main.cpp.obj  /out:_deps\recastnavigation-build\RecastBuilder\RecastBuilder.exe /implib:_deps\recastnavigation-build\RecastBuilder\RecastBuilder.lib /pdb:_deps\recastnavigation-build\RecastBuilder\RecastBuilder.pdb /version:0.0 /machine:x64 /INCREMENTAL:NO /subsystem:windows /SUBSYSTEM:CONSOLE  _deps\recastnavigation-build\DebugUtils\DebugUtils.lib  _deps\recastnavigation-build\Detour\Detour.lib  _deps\recastnavigation-build\DetourCrowd\DetourCrowd.lib  _deps\recastnavigation-build\DetourTileCache\DetourTileCache.lib  _deps\recastnavigation-build\Recast\Recast.lib  _deps\recastnavigation-build\Detour\Detour.lib  kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib && cd ."
LINK: command "C:\PROGRA~1\MIB055~1\2022\COMMUN~1\VC\Tools\MSVC\1440~1.338\bin\Hostx64\x64\link.exe /nologo _deps\recastnavigation-build\RecastBuilder\CMakeFiles\RecastBuilder.dir\Contrib\fastlz\fastlz.c.obj _deps\recastnavigation-build\RecastBuilder\CMakeFiles\RecastBuilder.dir\Source\ChunkyTriMesh.cpp.obj _deps\recastnavigation-build\RecastBuilder\CMakeFiles\RecastBuilder.dir\Source\InputGeom.cpp.obj _deps\recastnavigation-build\RecastBuilder\CMakeFiles\RecastBuilder.dir\Source\MeshLoaderObj.cpp.obj _deps\recastnavigation-build\RecastBuilder\CMakeFiles\RecastBuilder.dir\Source\PerfTimer.cpp.obj _deps\recastnavigation-build\RecastBuilder\CMakeFiles\RecastBuilder.dir\Source\Sample.cpp.obj _deps\recastnavigation-build\RecastBuilder\CMakeFiles\RecastBuilder.dir\Source\SampleInterfaces.cpp.obj _deps\recastnavigation-build\RecastBuilder\CMakeFiles\RecastBuilder.dir\Source\Sample_SoloMesh.cpp.obj _deps\recastnavigation-build\RecastBuilder\CMakeFiles\RecastBuilder.dir\Source\Sample_TileMesh.cpp.obj _deps\recastnavigation-build\RecastBuilder\CMakeFiles\RecastBuilder.dir\Source\main.cpp.obj /out:_deps\recastnavigation-build\RecastBuilder\RecastBuilder.exe /implib:_deps\recastnavigation-build\RecastBuilder\RecastBuilder.lib /pdb:_deps\recastnavigation-build\RecastBuilder\RecastBuilder.pdb /version:0.0 /machine:x64 /INCREMENTAL:NO /subsystem:windows /SUBSYSTEM:CONSOLE _deps\recastnavigation-build\DebugUtils\DebugUtils.lib _deps\recastnavigation-build\Detour\Detour.lib _deps\recastnavigation-build\DetourCrowd\DetourCrowd.lib _deps\recastnavigation-build\DetourTileCache\DetourTileCache.lib _deps\recastnavigation-build\Recast\Recast.lib _deps\recastnavigation-build\Detour\Detour.lib kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib /MANIFEST:EMBED,ID=1" failed (exit code 1120) with the following output:
Recast.lib(RecastMeshDetail.cpp.obj) : error LNK2001: 无法解析的外部符号 __GSHandlerCheck
DetourCrowd.lib(DetourLocalBoundary.cpp.obj) : error LNK2001: 无法解析的外部符号 __GSHandlerCheck
DetourCrowd.lib(DetourPathCorridor.cpp.obj) : error LNK2001: 无法解析的外部符号 __GSHandlerCheck
Recast.lib(Recast.cpp.obj) : error LNK2001: 无法解析的外部符号 __GSHandlerCheck
Recast.lib(RecastRasterization.cpp.obj) : error LNK2001: 无法解析的外部符号 __GSHandlerCheck
Detour.lib(DetourNavMesh.cpp.obj) : error LNK2001: 无法解析的外部符号 __GSHandlerCheck
Detour.lib(DetourNavMeshQuery.cpp.obj) : error LNK2001: 无法解析的外部符号 __GSHandlerCheck
DetourCrowd.lib(DetourCrowd.cpp.obj) : error LNK2001: 无法解析的外部符号 __GSHandlerCheck
DetourCrowd.lib(DetourObstacleAvoidance.cpp.obj) : error LNK2001: 无法解析的外部符号 __GSHandlerCheck
Sample_SoloMesh.cpp.obj : error LNK2001: 无法解析的外部符号 __GSHandlerCheck
Sample_TileMesh.cpp.obj : error LNK2001: 无法解析的外部符号 __GSHandlerCheck
DebugUtils.lib(DebugDraw.cpp.obj) : error LNK2001: 无法解析的外部符号 __GSHandlerCheck
DebugUtils.lib(RecastDump.cpp.obj) : error LNK2001: 无法解析的外部符号 __GSHandlerCheck
fastlz.c.obj : error LNK2001: 无法解析的外部符号 __GSHandlerCheck
InputGeom.cpp.obj : error LNK2001: 无法解析的外部符号 __GSHandlerCheck
MeshLoaderObj.cpp.obj : error LNK2001: 无法解析的外部符号 __GSHandlerCheck
Sample.cpp.obj : error LNK2001: 无法解析的外部符号 __GSHandlerCheck
fastlz.c.obj : error LNK2019: 无法解析的外部符号 __chkstk，函数 fastlz1_compress 中引用了该符号
main.cpp.obj : error LNK2001: 无法解析的外部符号 __chkstk
Recast.lib(RecastRasterization.cpp.obj) : error LNK2001: 无法解析的外部符号 __security_check_cookie
Recast.lib(RecastArea.cpp.obj) : error LNK2001: 无法解析的外部符号 __security_check_cookie
Recast.lib(RecastRegion.cpp.obj) : error LNK2001: 无法解析的外部符号 __security_check_cookie
Recast.lib(RecastMeshDetail.cpp.obj) : error LNK2001: 无法解析的外部符号 __security_check_cookie
DetourCrowd.lib(DetourObstacleAvoidance.cpp.obj) : error LNK2001: 无法解析的外部符号 __security_check_cookie
DetourCrowd.lib(DetourLocalBoundary.cpp.obj) : error LNK2001: 无法解析的外部符号 __security_check_cookie
DetourCrowd.lib(DetourPathCorridor.cpp.obj) : error LNK2001: 无法解析的外部符号 __security_check_cookie
Recast.lib(Recast.cpp.obj) : error LNK2001: 无法解析的外部符号 __security_check_cookie
DebugUtils.lib(RecastDump.cpp.obj) : error LNK2001: 无法解析的外部符号 __security_check_cookie
Detour.lib(DetourNavMesh.cpp.obj) : error LNK2001: 无法解析的外部符号 __security_check_cookie
Detour.lib(DetourNavMeshQuery.cpp.obj) : error LNK2001: 无法解析的外部符号 __security_check_cookie
DetourCrowd.lib(DetourCrowd.cpp.obj) : error LNK2001: 无法解析的外部符号 __security_check_cookie
Sample_SoloMesh.cpp.obj : error LNK2001: 无法解析的外部符号 __security_check_cookie
Sample_TileMesh.cpp.obj : error LNK2001: 无法解析的外部符号 __security_check_cookie
main.cpp.obj : error LNK2001: 无法解析的外部符号 __security_check_cookie
DebugUtils.lib(DebugDraw.cpp.obj) : error LNK2001: 无法解析的外部符号 __security_check_cookie
fastlz.c.obj : error LNK2001: 无法解析的外部符号 __security_check_cookie
InputGeom.cpp.obj : error LNK2001: 无法解析的外部符号 __security_check_cookie
MeshLoaderObj.cpp.obj : error LNK2001: 无法解析的外部符号 __security_check_cookie
Sample.cpp.obj : error LNK2001: 无法解析的外部符号 __security_check_cookie
Recast.lib(RecastRegion.cpp.obj) : error LNK2001: 无法解析的外部符号 memset
Recast.lib(RecastContour.cpp.obj) : error LNK2001: 无法解析的外部符号 memset
Recast.lib(RecastMesh.cpp.obj) : error LNK2001: 无法解析的外部符号 memset
Recast.lib(RecastMeshDetail.cpp.obj) : error LNK2001: 无法解析的外部符号 memset
DetourCrowd.lib(DetourObstacleAvoidance.cpp.obj) : error LNK2001: 无法解析的外部符号 memset
DetourCrowd.lib(DetourProximityGrid.cpp.obj) : error LNK2001: 无法解析的外部符号 memset
Recast.lib(Recast.cpp.obj) : error LNK2001: 无法解析的外部符号 memset
Recast.lib(RecastArea.cpp.obj) : error LNK2001: 无法解析的外部符号 memset
Detour.lib(DetourNavMesh.cpp.obj) : error LNK2001: 无法解析的外部符号 memset
Detour.lib(DetourNavMeshBuilder.cpp.obj) : error LNK2001: 无法解析的外部符号 memset
Detour.lib(DetourNode.cpp.obj) : error LNK2001: 无法解析的外部符号 memset
DetourCrowd.lib(DetourCrowd.cpp.obj) : error LNK2001: 无法解析的外部符号 memset
SampleInterfaces.cpp.obj : error LNK2001: 无法解析的外部符号 memset
Sample_SoloMesh.cpp.obj : error LNK2001: 无法解析的外部符号 memset
Sample_TileMesh.cpp.obj : error LNK2001: 无法解析的外部符号 memset
DebugUtils.lib(RecastDump.cpp.obj) : error LNK2001: 无法解析的外部符号 memset
fastlz.c.obj : error LNK2001: 无法解析的外部符号 memset
InputGeom.cpp.obj : error LNK2001: 无法解析的外部符号 memset
MeshLoaderObj.cpp.obj : error LNK2001: 无法解析的外部符号 memset
Sample.cpp.obj : error LNK2001: 无法解析的外部符号 memset
Recast.lib(RecastRasterization.cpp.obj) : error LNK2001: 无法解析的外部符号 __security_cookie
Recast.lib(RecastArea.cpp.obj) : error LNK2001: 无法解析的外部符号 __security_cookie
Recast.lib(RecastRegion.cpp.obj) : error LNK2001: 无法解析的外部符号 __security_cookie
Recast.lib(RecastMeshDetail.cpp.obj) : error LNK2001: 无法解析的外部符号 __security_cookie
DetourCrowd.lib(DetourObstacleAvoidance.cpp.obj) : error LNK2001: 无法解析的外部符号 __security_cookie
DetourCrowd.lib(DetourLocalBoundary.cpp.obj) : error LNK2001: 无法解析的外部符号 __security_cookie
DetourCrowd.lib(DetourPathCorridor.cpp.obj) : error LNK2001: 无法解析的外部符号 __security_cookie
Recast.lib(Recast.cpp.obj) : error LNK2001: 无法解析的外部符号 __security_cookie
DebugUtils.lib(RecastDump.cpp.obj) : error LNK2001: 无法解析的外部符号 __security_cookie
Detour.lib(DetourNavMesh.cpp.obj) : error LNK2001: 无法解析的外部符号 __security_cookie
Detour.lib(DetourNavMeshQuery.cpp.obj) : error LNK2001: 无法解析的外部符号 __security_cookie
DetourCrowd.lib(DetourCrowd.cpp.obj) : error LNK2001: 无法解析的外部符号 __security_cookie
Sample_SoloMesh.cpp.obj : error LNK2001: 无法解析的外部符号 __security_cookie
Sample_TileMesh.cpp.obj : error LNK2001: 无法解析的外部符号 __security_cookie
main.cpp.obj : error LNK2001: 无法解析的外部符号 __security_cookie
DebugUtils.lib(DebugDraw.cpp.obj) : error LNK2001: 无法解析的外部符号 __security_cookie
fastlz.c.obj : error LNK2001: 无法解析的外部符号 __security_cookie
InputGeom.cpp.obj : error LNK2001: 无法解析的外部符号 __security_cookie
MeshLoaderObj.cpp.obj : error LNK2001: 无法解析的外部符号 __security_cookie
Sample.cpp.obj : error LNK2001: 无法解析的外部符号 __security_cookie
Sample_TileMesh.cpp.obj : error LNK2001: 无法解析的外部符号 "void * __cdecl operator new[](unsigned __int64)" (??_U@YAPEAX_K@Z)
DebugUtils.lib(DebugDraw.cpp.obj) : error LNK2001: 无法解析的外部符号 "void * __cdecl operator new[](unsigned __int64)" (??_U@YAPEAX_K@Z)
ChunkyTriMesh.cpp.obj : error LNK2001: 无法解析的外部符号 "void * __cdecl operator new[](unsigned __int64)" (??_U@YAPEAX_K@Z)
InputGeom.cpp.obj : error LNK2001: 无法解析的外部符号 "void * __cdecl operator new[](unsigned __int64)" (??_U@YAPEAX_K@Z)
MeshLoaderObj.cpp.obj : error LNK2001: 无法解析的外部符号 "void * __cdecl operator new[](unsigned __int64)" (??_U@YAPEAX_K@Z)
Sample_SoloMesh.cpp.obj : error LNK2001: 无法解析的外部符号 "void * __cdecl operator new[](unsigned __int64)" (??_U@YAPEAX_K@Z)
Sample_TileMesh.cpp.obj : error LNK2001: 无法解析的外部符号 "void __cdecl operator delete[](void *)" (??_V@YAXPEAX@Z)
DebugUtils.lib(DebugDraw.cpp.obj) : error LNK2001: 无法解析的外部符号 "void __cdecl operator delete[](void *)" (??_V@YAXPEAX@Z)
ChunkyTriMesh.cpp.obj : error LNK2001: 无法解析的外部符号 "void __cdecl operator delete[](void *)" (??_V@YAXPEAX@Z)
InputGeom.cpp.obj : error LNK2001: 无法解析的外部符号 "void __cdecl operator delete[](void *)" (??_V@YAXPEAX@Z)
MeshLoaderObj.cpp.obj : error LNK2001: 无法解析的外部符号 "void __cdecl operator delete[](void *)" (??_V@YAXPEAX@Z)
Sample_SoloMesh.cpp.obj : error LNK2001: 无法解析的外部符号 "void __cdecl operator delete[](void *)" (??_V@YAXPEAX@Z)
ChunkyTriMesh.cpp.obj : error LNK2019: 无法解析的外部符号 __imp_qsort，函数 "void __cdecl subdivide(struct BoundsItem *,int,int,int,int,int &,struct rcChunkyTriMeshNode *,int,int &,int *,int const *)" (?subdivide@@YAXPEAUBoundsItem@@HHHHAEAHPEAUrcChunkyTriMeshNode@@H1PEAHPEBH@Z) 中引用了该符号
Detour.lib(DetourNavMeshBuilder.cpp.obj) : error LNK2001: 无法解析的外部符号 __imp_qsort
Recast.lib(RecastContour.cpp.obj) : error LNK2001: 无法解析的外部符号 __imp_qsort
Recast.lib(RecastMesh.cpp.obj) : error LNK2001: 无法解析的外部符号 _fltused
Recast.lib(RecastMeshDetail.cpp.obj) : error LNK2001: 无法解析的外部符号 _fltused
Recast.lib(Recast.cpp.obj) : error LNK2001: 无法解析的外部符号 _fltused
Recast.lib(RecastRasterization.cpp.obj) : error LNK2001: 无法解析的外部符号 _fltused
Recast.lib(RecastArea.cpp.obj) : error LNK2001: 无法解析的外部符号 _fltused
Recast.lib(RecastContour.cpp.obj) : error LNK2001: 无法解析的外部符号 _fltused
DetourCrowd.lib(DetourLocalBoundary.cpp.obj) : error LNK2001: 无法解析的外部符号 _fltused
DetourCrowd.lib(DetourPathCorridor.cpp.obj) : error LNK2001: 无法解析的外部符号 _fltused
DetourCrowd.lib(DetourProximityGrid.cpp.obj) : error LNK2001: 无法解析的外部符号 _fltused
DetourCrowd.lib(DetourPathQueue.cpp.obj) : error LNK2001: 无法解析的外部符号 _fltused
Detour.lib(DetourCommon.cpp.obj) : error LNK2001: 无法解析的外部符号 _fltused
Detour.lib(DetourNode.cpp.obj) : error LNK2001: 无法解析的外部符号 _fltused
DetourCrowd.lib(DetourCrowd.cpp.obj) : error LNK2001: 无法解析的外部符号 _fltused
DetourCrowd.lib(DetourObstacleAvoidance.cpp.obj) : error LNK2001: 无法解析的外部符号 _fltused
DebugUtils.lib(RecastDump.cpp.obj) : error LNK2001: 无法解析的外部符号 _fltused
Detour.lib(DetourNavMesh.cpp.obj) : error LNK2001: 无法解析的外部符号 _fltused
Detour.lib(DetourNavMeshQuery.cpp.obj) : error LNK2001: 无法解析的外部符号 _fltused
Detour.lib(DetourNavMeshBuilder.cpp.obj) : error LNK2001: 无法解析的外部符号 _fltused
Sample_SoloMesh.cpp.obj : error LNK2001: 无法解析的外部符号 _fltused
Sample_TileMesh.cpp.obj : error LNK2001: 无法解析的外部符号 _fltused
main.cpp.obj : error LNK2001: 无法解析的外部符号 _fltused
DebugUtils.lib(DebugDraw.cpp.obj) : error LNK2001: 无法解析的外部符号 _fltused
ChunkyTriMesh.cpp.obj : error LNK2001: 无法解析的外部符号 _fltused
InputGeom.cpp.obj : error LNK2001: 无法解析的外部符号 _fltused
MeshLoaderObj.cpp.obj : error LNK2001: 无法解析的外部符号 _fltused
Sample.cpp.obj : error LNK2001: 无法解析的外部符号 _fltused
InputGeom.cpp.obj : error LNK2019: 无法解析的外部符号 "void * __cdecl operator new(unsigned __int64)" (??2@YAPEAX_K@Z)，函数 "void * __cdecl std::_Allocate<16,struct std::_Default_allocate_traits,0>(unsigned __int64)" (??$_Allocate@$0BA@U_Default_allocate_traits@std@@$0A@@std@@YAPEAX_K@Z) 中引用了该符号
MeshLoaderObj.cpp.obj : error LNK2001: 无法解析的外部符号 "void * __cdecl operator new(unsigned __int64)" (??2@YAPEAX_K@Z)
main.cpp.obj : error LNK2001: 无法解析的外部符号 "void * __cdecl operator new(unsigned __int64)" (??2@YAPEAX_K@Z)
DetourCrowd.lib(DetourObstacleAvoidance.cpp.obj) : error LNK2001: 无法解析的外部符号 "void __cdecl operator delete(void *,unsigned __int64)" (??3@YAXPEAX_K@Z)
DetourCrowd.lib(DetourProximityGrid.cpp.obj) : error LNK2001: 无法解析的外部符号 "void __cdecl operator delete(void *,unsigned __int64)" (??3@YAXPEAX_K@Z)
Recast.lib(Recast.cpp.obj) : error LNK2001: 无法解析的外部符号 "void __cdecl operator delete(void *,unsigned __int64)" (??3@YAXPEAX_K@Z)
Recast.lib(RecastRegion.cpp.obj) : error LNK2001: 无法解析的外部符号 "void __cdecl operator delete(void *,unsigned __int64)" (??3@YAXPEAX_K@Z)
DebugUtils.lib(RecastDump.cpp.obj) : error LNK2001: 无法解析的外部符号 "void __cdecl operator delete(void *,unsigned __int64)" (??3@YAXPEAX_K@Z)
Detour.lib(DetourNavMesh.cpp.obj) : error LNK2001: 无法解析的外部符号 "void __cdecl operator delete(void *,unsigned __int64)" (??3@YAXPEAX_K@Z)
Detour.lib(DetourNavMeshQuery.cpp.obj) : error LNK2001: 无法解析的外部符号 "void __cdecl operator delete(void *,unsigned __int64)" (??3@YAXPEAX_K@Z)
DetourCrowd.lib(DetourCrowd.cpp.obj) : error LNK2001: 无法解析的外部符号 "void __cdecl operator delete(void *,unsigned __int64)" (??3@YAXPEAX_K@Z)
Sample_SoloMesh.cpp.obj : error LNK2001: 无法解析的外部符号 "void __cdecl operator delete(void *,unsigned __int64)" (??3@YAXPEAX_K@Z)
Sample_TileMesh.cpp.obj : error LNK2001: 无法解析的外部符号 "void __cdecl operator delete(void *,unsigned __int64)" (??3@YAXPEAX_K@Z)
main.cpp.obj : error LNK2001: 无法解析的外部符号 "void __cdecl operator delete(void *,unsigned __int64)" (??3@YAXPEAX_K@Z)
DebugUtils.lib(DebugDraw.cpp.obj) : error LNK2001: 无法解析的外部符号 "void __cdecl operator delete(void *,unsigned __int64)" (??3@YAXPEAX_K@Z)
InputGeom.cpp.obj : error LNK2001: 无法解析的外部符号 "void __cdecl operator delete(void *,unsigned __int64)" (??3@YAXPEAX_K@Z)
MeshLoaderObj.cpp.obj : error LNK2001: 无法解析的外部符号 "void __cdecl operator delete(void *,unsigned __int64)" (??3@YAXPEAX_K@Z)
Sample.cpp.obj : error LNK2001: 无法解析的外部符号 "void __cdecl operator delete(void *,unsigned __int64)" (??3@YAXPEAX_K@Z)
SampleInterfaces.cpp.obj : error LNK2001: 无法解析的外部符号 "void __cdecl operator delete(void *,unsigned __int64)" (??3@YAXPEAX_K@Z)
InputGeom.cpp.obj : error LNK2019: 无法解析的外部符号 __imp__invalid_parameter_noinfo_noreturn，函数 "void * __cdecl std::_Allocate<16,struct std::_Default_allocate_traits,0>(unsigned __int64)" (??$_Allocate@$0BA@U_Default_allocate_traits@std@@$0A@@std@@YAPEAX_K@Z) 中引用了该符号
MeshLoaderObj.cpp.obj : error LNK2001: 无法解析的外部符号 __imp__invalid_parameter_noinfo_noreturn
main.cpp.obj : error LNK2001: 无法解析的外部符号 __imp__invalid_parameter_noinfo_noreturn
InputGeom.cpp.obj : error LNK2019: 无法解析的外部符号 __imp_fclose，函数 "private: bool __cdecl InputGeom::loadGeomSet(class rcContext *,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &)" (?loadGeomSet@InputGeom@@AEAA_NPEAVrcContext@@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@@Z) 中引用了该符号
MeshLoaderObj.cpp.obj : error LNK2001: 无法解析的外部符号 __imp_fclose
Sample.cpp.obj : error LNK2001: 无法解析的外部符号 __imp_fclose
InputGeom.cpp.obj : error LNK2019: 无法解析的外部符号 __imp_fopen，函数 "private: bool __cdecl InputGeom::loadGeomSet(class rcContext *,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &)" (?loadGeomSet@InputGeom@@AEAA_NPEAVrcContext@@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@@Z) 中引用了该 符号
MeshLoaderObj.cpp.obj : error LNK2001: 无法解析的外部符号 __imp_fopen
Sample.cpp.obj : error LNK2001: 无法解析的外部符号 __imp_fopen
InputGeom.cpp.obj : error LNK2019: 无法解析的外部符号 __imp_fread，函数 "private: bool __cdecl InputGeom::loadGeomSet(class rcContext *,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &)" (?loadGeomSet@InputGeom@@AEAA_NPEAVrcContext@@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@@Z) 中引用了该 符号
MeshLoaderObj.cpp.obj : error LNK2001: 无法解析的外部符号 __imp_fread
Sample.cpp.obj : error LNK2001: 无法解析的外部符号 __imp_fread
InputGeom.cpp.obj : error LNK2019: 无法解析的外部符号 __imp_fseek，函数 "private: bool __cdecl InputGeom::loadGeomSet(class rcContext *,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &)" (?loadGeomSet@InputGeom@@AEAA_NPEAVrcContext@@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@@Z) 中引用了该 符号
MeshLoaderObj.cpp.obj : error LNK2001: 无法解析的外部符号 __imp_fseek
InputGeom.cpp.obj : error LNK2019: 无法解析的外部符号 __imp_ftell，函数 "private: bool __cdecl InputGeom::loadGeomSet(class rcContext *,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &)" (?loadGeomSet@InputGeom@@AEAA_NPEAVrcContext@@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@@Z) 中引用了该 符号
MeshLoaderObj.cpp.obj : error LNK2001: 无法解析的外部符号 __imp_ftell
Sample_TileMesh.cpp.obj : error LNK2001: 无法解析的外部符号 __imp___stdio_common_vfprintf
main.cpp.obj : error LNK2001: 无法解析的外部符号 __imp___stdio_common_vfprintf
DebugUtils.lib(RecastDump.cpp.obj) : error LNK2001: 无法解析的外部符号 __imp___stdio_common_vfprintf
InputGeom.cpp.obj : error LNK2001: 无法解析的外部符号 __imp___stdio_common_vfprintf
Sample.cpp.obj : error LNK2001: 无法解析的外部符号 __imp___stdio_common_vfprintf
SampleInterfaces.cpp.obj : error LNK2001: 无法解析的外部符号 __imp___stdio_common_vfprintf
Sample_SoloMesh.cpp.obj : error LNK2001: 无法解析的外部符号 __imp___stdio_common_vfprintf
InputGeom.cpp.obj : error LNK2019: 无法解析的外部符号 __imp___stdio_common_vsscanf，函数 _vsscanf_l 中引用了该符号
MeshLoaderObj.cpp.obj : error LNK2001: 无法解析的外部符号 __imp___stdio_common_vsscanf
InputGeom.cpp.obj : error LNK2019: 无法解析的外部符号 __imp_isspace，函数 "private: bool __cdecl InputGeom::loadGeomSet(class rcContext *,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &)" (?loadGeomSet@InputGeom@@AEAA_NPEAVrcContext@@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@@Z) 中引用了 该符号
InputGeom.cpp.obj : error LNK2019: 无法解析的外部符号 __imp_tolower，函数 "public: bool __cdecl InputGeom::load(class rcContext *,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &)" (?load@InputGeom@@QEAA_NPEAVrcContext@@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@@Z) 中引用了该符号
InputGeom.cpp.obj : error LNK2019: 无法解析的外部符号 __std_exception_copy，函数 "public: __cdecl std::bad_alloc::bad_alloc(class std::bad_alloc const &)" (??0bad_alloc@std@@QEAA@AEBV01@@Z) 中引用了该符号
MeshLoaderObj.cpp.obj : error LNK2001: 无法解析的外部符号 __std_exception_copy
main.cpp.obj : error LNK2001: 无法解析的外部符号 __std_exception_copy
InputGeom.cpp.obj : error LNK2019: 无法解析的外部符号 __std_exception_destroy，函数 "public: virtual __cdecl std::bad_alloc::~bad_alloc(void)" (??1bad_alloc@std@@UEAA@XZ) 中引用了该符号
MeshLoaderObj.cpp.obj : error LNK2001: 无法解析的外部符号 __std_exception_destroy
main.cpp.obj : error LNK2001: 无法解析的外部符号 __std_exception_destroy
InputGeom.cpp.obj : error LNK2019: 无法解析的外部符号 "void __cdecl std::_Xlength_error(char const *)" (?_Xlength_error@std@@YAXPEBD@Z)，函数 "void __cdecl std::_Xlen_string(void)" (?_Xlen_string@std@@YAXXZ) 中引用了该符号
MeshLoaderObj.cpp.obj : error LNK2001: 无法解析的外部符号 "void __cdecl std::_Xlength_error(char const *)" (?_Xlength_error@std@@YAXPEBD@Z)
main.cpp.obj : error LNK2001: 无法解析的外部符号 "void __cdecl std::_Xlength_error(char const *)" (?_Xlength_error@std@@YAXPEBD@Z)
InputGeom.cpp.obj : error LNK2019: 无法解析的外部符号 "void __cdecl std::_Xout_of_range(char const *)" (?_Xout_of_range@std@@YAXPEBD@Z)，函数 "public: static void __cdecl std::_String_val<struct std::_Simple_types<char> >::_Xran(void)" (?_Xran@?$_String_val@U?$_Simple_types@D@std@@@std@@SAXXZ) 中引用了该符号
main.cpp.obj : error LNK2001: 无法解析的外部符号 "void __cdecl std::_Xout_of_range(char const *)" (?_Xout_of_range@std@@YAXPEBD@Z)
InputGeom.cpp.obj : error LNK2019: 无法解析的外部符号 _CxxThrowException，函数 "void __cdecl std::_Throw_bad_array_new_length(void)" (?_Throw_bad_array_new_length@std@@YAXXZ) 中引用了该符号
MeshLoaderObj.cpp.obj : error LNK2001: 无法解析的外部符号 _CxxThrowException
main.cpp.obj : error LNK2001: 无法解析的外部符号 _CxxThrowException
Recast.lib(RecastMesh.cpp.obj) : error LNK2019: 无法解析的外部符号 __CxxFrameHandler4，函数 "bool __cdecl rcBuildPolyMesh(class rcContext *,struct rcContourSet const &,int,struct rcPolyMesh &)" (?rcBuildPolyMesh@@YA_NPEAVrcContext@@AEBUrcContourSet@@HAEAUrcPolyMesh@@@Z) 中引用了该符号
Recast.lib(RecastMeshDetail.cpp.obj) : error LNK2001: 无法解析的外部符号 __CxxFrameHandler4
Recast.lib(RecastFilter.cpp.obj) : error LNK2001: 无法解析的外部符号 __CxxFrameHandler4
Recast.lib(RecastArea.cpp.obj) : error LNK2001: 无法解析的外部符号 __CxxFrameHandler4
Recast.lib(RecastRegion.cpp.obj) : error LNK2001: 无法解析的外部符号 __CxxFrameHandler4
Recast.lib(RecastContour.cpp.obj) : error LNK2001: 无法解析的外部符号 __CxxFrameHandler4
DetourCrowd.lib(DetourProximityGrid.cpp.obj) : error LNK2001: 无法解析的外部符号 __CxxFrameHandler4
DetourCrowd.lib(DetourPathQueue.cpp.obj) : error LNK2001: 无法解析的外部符号 __CxxFrameHandler4
Recast.lib(Recast.cpp.obj) : error LNK2001: 无法解析的外部符号 __CxxFrameHandler4
Recast.lib(RecastRasterization.cpp.obj) : error LNK2001: 无法解析的外部符号 __CxxFrameHandler4
Detour.lib(DetourNode.cpp.obj) : error LNK2001: 无法解析的外部符号 __CxxFrameHandler4
DetourCrowd.lib(DetourCrowd.cpp.obj) : error LNK2001: 无法解析的外部符号 __CxxFrameHandler4
DetourCrowd.lib(DetourObstacleAvoidance.cpp.obj) : error LNK2001: 无法解析的外部符号 __CxxFrameHandler4
DetourCrowd.lib(DetourPathCorridor.cpp.obj) : error LNK2001: 无法解析的外部符号 __CxxFrameHandler4
Sample_TileMesh.cpp.obj : error LNK2001: 无法解析的外部符号 __CxxFrameHandler4
DebugUtils.lib(DebugDraw.cpp.obj) : error LNK2001: 无法解析的外部符号 __CxxFrameHandler4
Detour.lib(DetourNavMesh.cpp.obj) : error LNK2001: 无法解析的外部符号 __CxxFrameHandler4
Detour.lib(DetourNavMeshQuery.cpp.obj) : error LNK2001: 无法解析的外部符号 __CxxFrameHandler4
InputGeom.cpp.obj : error LNK2001: 无法解析的外部符号 __CxxFrameHandler4
Sample.cpp.obj : error LNK2001: 无法解析的外部符号 __CxxFrameHandler4
SampleInterfaces.cpp.obj : error LNK2001: 无法解析的外部符号 __CxxFrameHandler4
Sample_SoloMesh.cpp.obj : error LNK2001: 无法解析的外部符号 __CxxFrameHandler4
Recast.lib(RecastRegion.cpp.obj) : error LNK2001: 无法解析的外部符号 __GSHandlerCheck_EH4
Recast.lib(RecastMeshDetail.cpp.obj) : error LNK2001: 无法解析的外部符号 __GSHandlerCheck_EH4
InputGeom.cpp.obj : error LNK2001: 无法解析的外部符号 __GSHandlerCheck_EH4
main.cpp.obj : error LNK2001: 无法解析的外部符号 __GSHandlerCheck_EH4
Detour.lib(DetourNavMeshQuery.cpp.obj) : error LNK2001: 无法解析的外部符号 __GSHandlerCheck_EH4
Recast.lib(RecastArea.cpp.obj) : error LNK2001: 无法解析的外部符号 __GSHandlerCheck_EH4
InputGeom.cpp.obj : error LNK2019: 无法解析的外部符号 memcmp，函数 "bool __cdecl std::operator==<char,struct std::char_traits<char>,class std::allocator<char> >(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,char const * const)" (??$?8DU?$char_traits@D@std@@V?$allocator@D@1@@std@@YA_NAEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@0@QEBD@Z) 中引用了该符号
main.cpp.obj : error LNK2001: 无法解析的外部符号 memcmp
Recast.lib(RecastMesh.cpp.obj) : error LNK2001: 无法解析的外部符号 memcpy
Recast.lib(RecastMeshDetail.cpp.obj) : error LNK2001: 无法解析的外部符号 memcpy
DetourCrowd.lib(DetourPathCorridor.cpp.obj) : error LNK2001: 无法解析的外部符号 memcpy
DetourCrowd.lib(DetourPathQueue.cpp.obj) : error LNK2001: 无法解析的外部符号 memcpy
Recast.lib(RecastArea.cpp.obj) : error LNK2001: 无法解析的外部符号 memcpy
Recast.lib(RecastContour.cpp.obj) : error LNK2001: 无法解析的外部符号 memcpy
DebugUtils.lib(DebugDraw.cpp.obj) : error LNK2001: 无法解析的外部符号 memcpy
Detour.lib(DetourNavMeshQuery.cpp.obj) : error LNK2001: 无法解析的外部符号 memcpy
Detour.lib(DetourNavMeshBuilder.cpp.obj) : error LNK2001: 无法解析的外部符号 memcpy
DetourCrowd.lib(DetourCrowd.cpp.obj) : error LNK2001: 无法解析的外部符号 memcpy
InputGeom.cpp.obj : error LNK2001: 无法解析的外部符号 memcpy
MeshLoaderObj.cpp.obj : error LNK2001: 无法解析的外部符号 memcpy
SampleInterfaces.cpp.obj : error LNK2001: 无法解析的外部符号 memcpy
main.cpp.obj : error LNK2001: 无法解析的外部符号 memcpy
DetourCrowd.lib(DetourCrowd.cpp.obj) : error LNK2001: 无法解析的外部符号 memmove
DetourCrowd.lib(DetourLocalBoundary.cpp.obj) : error LNK2001: 无法解析的外部符号 memmove
DetourCrowd.lib(DetourPathCorridor.cpp.obj) : error LNK2001: 无法解析的外部符号 memmove
Recast.lib(RecastMeshDetail.cpp.obj) : error LNK2001: 无法解析的外部符号 memmove
InputGeom.cpp.obj : error LNK2001: 无法解析的外部符号 memmove
MeshLoaderObj.cpp.obj : error LNK2001: 无法解析的外部符号 memmove
main.cpp.obj : error LNK2001: 无法解析的外部符号 memmove
Detour.lib(DetourNavMeshQuery.cpp.obj) : error LNK2001: 无法解析的外部符号 memmove
DebugUtils.lib(RecastDump.cpp.obj) : error LNK2001: 无法解析的外部符号 "const type_info::`vftable'" (??_7type_info@@6B@)
Detour.lib(DetourNavMeshQuery.cpp.obj) : error LNK2001: 无法解析的外部符号 "const type_info::`vftable'" (??_7type_info@@6B@)
Sample_SoloMesh.cpp.obj : error LNK2001: 无法解析的外部符号 "const type_info::`vftable'" (??_7type_info@@6B@)
Sample_TileMesh.cpp.obj : error LNK2001: 无法解析的外部符号 "const type_info::`vftable'" (??_7type_info@@6B@)
main.cpp.obj : error LNK2001: 无法解析的外部符号 "const type_info::`vftable'" (??_7type_info@@6B@)
DebugUtils.lib(DebugDraw.cpp.obj) : error LNK2001: 无法解析的外部符号 "const type_info::`vftable'" (??_7type_info@@6B@)
InputGeom.cpp.obj : error LNK2001: 无法解析的外部符号 "const type_info::`vftable'" (??_7type_info@@6B@)
MeshLoaderObj.cpp.obj : error LNK2001: 无法解析的外部符号 "const type_info::`vftable'" (??_7type_info@@6B@)
Sample.cpp.obj : error LNK2001: 无法解析的外部符号 "const type_info::`vftable'" (??_7type_info@@6B@)
SampleInterfaces.cpp.obj : error LNK2001: 无法解析的外部符号 "const type_info::`vftable'" (??_7type_info@@6B@)
MeshLoaderObj.cpp.obj : error LNK2019: 无法解析的外部符号 __imp_atoi，函数 "public: bool __cdecl rcMeshLoaderObj::load(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &)" (?load@rcMeshLoaderObj@@QEAA_NAEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@@Z) 中引用了该符号
Recast.lib(RecastMeshDetail.cpp.obj) : error LNK2019: 无法解析的外部符号 sqrtf，函数 "void __cdecl getHeightData(class rcContext *,struct rcCompactHeightfield const &,unsigned short const *,int,unsigned short const *,int,struct rcHeightPatch &,class rcIntArray &,int)" (?getHeightData@@YAXPEAVrcContext@@AEBUrcCompactHeightfield@@PEBGH2HAEAUrcHeightPatch@@AEAVrcIntArray@@H@Z) 中引用了该符号
DetourCrowd.lib(DetourCrowd.cpp.obj) : error LNK2001: 无法解析的外部符号 sqrtf
DetourCrowd.lib(DetourObstacleAvoidance.cpp.obj) : error LNK2001: 无法解析的外部符号 sqrtf
DetourCrowd.lib(DetourPathCorridor.cpp.obj) : error LNK2001: 无法解析的外部符号 sqrtf
Recast.lib(Recast.cpp.obj) : error LNK2001: 无法解析的外部符号 sqrtf
MeshLoaderObj.cpp.obj : error LNK2001: 无法解析的外部符号 sqrtf
DebugUtils.lib(DebugDraw.cpp.obj) : error LNK2001: 无法解析的外部符号 sqrtf
Detour.lib(DetourNavMeshQuery.cpp.obj) : error LNK2001: 无法解析的外部符号 sqrtf
Detour.lib(DetourCommon.cpp.obj) : error LNK2001: 无法解析的外部符号 sqrtf
MeshLoaderObj.cpp.obj : error LNK2019: 无法解析的外部符号 strcmp，函数 "public: bool __cdecl rcMeshLoaderObj::load(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &)" (?load@rcMeshLoaderObj@@QEAA_NAEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@@Z) 中引用了该符号
PerfTimer.cpp.obj : error LNK2019: 无法解析的外部符号 __imp_QueryPerformanceCounter，函数 "__int64 __cdecl getPerfTime(void)" (?getPerfTime@@YA_JXZ) 中引用了该符号
PerfTimer.cpp.obj : error LNK2019: 无法解析的外部符号 __imp_QueryPerformanceFrequency，函数 "int __cdecl getPerfTimeUsec(__int64)" (?getPerfTimeUsec@@YAH_J@Z) 中引用了该符号
Recast.lib(RecastRegion.cpp.obj) : error LNK2001: 无法解析的外部符号 __std_terminate
Recast.lib(RecastContour.cpp.obj) : error LNK2001: 无法解析的外部符号 __std_terminate
Recast.lib(RecastMesh.cpp.obj) : error LNK2001: 无法解析的外部符号 __std_terminate
Recast.lib(RecastMeshDetail.cpp.obj) : error LNK2001: 无法解析的外部符号 __std_terminate
Recast.lib(Recast.cpp.obj) : error LNK2001: 无法解析的外部符号 __std_terminate
Recast.lib(RecastRasterization.cpp.obj) : error LNK2001: 无法解析的外部符号 __std_terminate
Recast.lib(RecastFilter.cpp.obj) : error LNK2001: 无法解析的外部符号 __std_terminate
Recast.lib(RecastArea.cpp.obj) : error LNK2001: 无法解析的外部符号 __std_terminate
Detour.lib(DetourNavMeshQuery.cpp.obj) : error LNK2001: 无法解析的外部符号 __std_terminate
DetourCrowd.lib(DetourCrowd.cpp.obj) : error LNK2001: 无法解析的外部符号 __std_terminate
DetourCrowd.lib(DetourObstacleAvoidance.cpp.obj) : error LNK2001: 无法解析的外部符号 __std_terminate
DetourCrowd.lib(DetourProximityGrid.cpp.obj) : error LNK2001: 无法解析的外部符号 __std_terminate
Sample.cpp.obj : error LNK2001: 无法解析的外部符号 __std_terminate
Sample_SoloMesh.cpp.obj : error LNK2001: 无法解析的外部符号 __std_terminate
Sample_TileMesh.cpp.obj : error LNK2001: 无法解析的外部符号 __std_terminate
Detour.lib(DetourNavMesh.cpp.obj) : error LNK2001: 无法解析的外部符号 __std_terminate
main.cpp.obj : error LNK2001: 无法解析的外部符号 __imp___acrt_iob_func
DebugUtils.lib(RecastDump.cpp.obj) : error LNK2001: 无法解析的外部符号 __imp___acrt_iob_func
Sample.cpp.obj : error LNK2001: 无法解析的外部符号 __imp___acrt_iob_func
SampleInterfaces.cpp.obj : error LNK2001: 无法解析的外部符号 __imp___acrt_iob_func
Sample_SoloMesh.cpp.obj : error LNK2001: 无法解析的外部符号 __imp___acrt_iob_func
Sample_TileMesh.cpp.obj : error LNK2001: 无法解析的外部符号 __imp___acrt_iob_func
Sample.cpp.obj : error LNK2019: 无法解析的外部符号 __imp_fwrite，函数 "public: void __cdecl Sample::saveAll(char const *,class dtNavMesh const *)" (?saveAll@Sample@@QEAAXPEBDPEBVdtNavMesh@@@Z) 中引用了该符号
SampleInterfaces.cpp.obj : error LNK2019: 无法解析的外部符号 __imp_putchar，函数 "public: void __cdecl BuildContext::dumpLog(char const *,...)" (?dumpLog@BuildContext@@QEAAXPEBDZZ) 中引用了该符号
Recast.lib(RecastMeshDetail.cpp.obj) : error LNK2001: 无法解析的外部符号 ceilf
Sample_SoloMesh.cpp.obj : error LNK2001: 无法解析的外部符号 ceilf
Sample_TileMesh.cpp.obj : error LNK2001: 无法解析的外部符号 ceilf
Detour.lib(DetourNavMeshBuilder.cpp.obj) : error LNK2001: 无法解析的外部符号 ceilf
Recast.lib(RecastRasterization.cpp.obj) : error LNK2001: 无法解析的外部符号 ceilf
DetourCrowd.lib(DetourProximityGrid.cpp.obj) : error LNK2001: 无法解析的外部符号 floorf
Recast.lib(RecastRasterization.cpp.obj) : error LNK2001: 无法解析的外部符号 floorf
Recast.lib(RecastMesh.cpp.obj) : error LNK2001: 无法解析的外部符号 floorf
Recast.lib(RecastMeshDetail.cpp.obj) : error LNK2001: 无法解析的外部符号 floorf
Sample_SoloMesh.cpp.obj : error LNK2001: 无法解析的外部符号 floorf
Sample_TileMesh.cpp.obj : error LNK2001: 无法解析的外部符号 floorf
Detour.lib(DetourNavMesh.cpp.obj) : error LNK2001: 无法解析的外部符号 floorf
Detour.lib(DetourNavMeshBuilder.cpp.obj) : error LNK2001: 无法解析的外部符号 floorf
main.cpp.obj : error LNK2019: 无法解析的外部符号 __imp_atof，函数 main 中引用了该符号
LINK : error LNK2001: 无法解析的外部符号 mainCRTStartup
DebugUtils.lib(DebugDraw.cpp.obj) : error LNK2001: 无法解析的外部符号 _purecall
DebugUtils.lib(RecastDump.cpp.obj) : error LNK2001: 无法解析的外部符号 _purecall
Detour.lib(DetourNavMeshQuery.cpp.obj) : error LNK2001: 无法解析的外部符号 _purecall
DebugUtils.lib(DebugDraw.cpp.obj) : error LNK2019: 无法解析的外部符号 cosf，函数 "float __cdecl dtMathCosf(float)" (?dtMathCosf@@YAMM@Z) 中引用了该符号
DetourCrowd.lib(DetourObstacleAvoidance.cpp.obj) : error LNK2001: 无法解析的外部符号 cosf
Recast.lib(Recast.cpp.obj) : error LNK2001: 无法解析的外部符号 cosf
DebugUtils.lib(DebugDraw.cpp.obj) : error LNK2019: 无法解析的外部符号 sinf，函数 "float __cdecl dtMathSinf(float)" (?dtMathSinf@@YAMM@Z) 中引用了该符号
DetourCrowd.lib(DetourObstacleAvoidance.cpp.obj) : error LNK2001: 无法解析的外部符号 sinf
DebugUtils.lib(RecastDump.cpp.obj) : error LNK2019: 无法解析的外部符号 __imp___stdio_common_vsprintf，函数 "void __cdecl ioprintf(struct duFileIO *,char const *,...)" (?ioprintf@@YAXPEAUduFileIO@@PEBDZZ) 中引用了该符号
Recast.lib(Recast.cpp.obj) : error LNK2001: 无法解析的外部符号 __imp___stdio_common_vsprintf
Detour.lib(DetourAlloc.cpp.obj) : error LNK2019: 无法解析的外部符号 __imp_free，函数 "void __cdecl dtFreeDefault(void *)" (?dtFreeDefault@@YAXPEAX@Z) 中引用了该符号
Recast.lib(RecastAlloc.cpp.obj) : error LNK2001: 无法解析的外部符号 __imp_free
Detour.lib(DetourAlloc.cpp.obj) : error LNK2019: 无法解析的外部符号 __imp_malloc，函数 "void * __cdecl dtAllocDefault(unsigned __int64,enum dtAllocHint)" (?dtAllocDefault@@YAPEAX_KW4dtAllocHint@@@Z) 中引用了该符号
Recast.lib(RecastAlloc.cpp.obj) : error LNK2001: 无法解析的外部符号 __imp_malloc
Detour.lib(DetourNavMesh.cpp.obj) : error LNK2019: 无法解析的外部符号 __imp_roundf，函数 "private: void __cdecl dtNavMesh::connectExtLinks(struct dtMeshTile *,struct dtMeshTile *,int)" (?connectExtLinks@dtNavMesh@@AEAAXPEAUdtMeshTile@@0H@Z) 中引用了该符号
Detour.lib(DetourNavMeshQuery.cpp.obj) : error LNK2019: 无法解析的外部符号 _Init_thread_header，函数 "bool __cdecl dtVequal(float const *,float const *)" (?dtVequal@@YA_NPEBM0@Z) 中引用了该符号
Detour.lib(DetourNavMeshQuery.cpp.obj) : error LNK2019: 无法解析的外部符号 _Init_thread_footer，函数 "bool __cdecl dtVequal(float const *,float const *)" (?dtVequal@@YA_NPEBM0@Z) 中引用了该符号
Detour.lib(DetourNavMeshQuery.cpp.obj) : error LNK2019: 无法解析的外部符号 _fdclass，函数 "int __cdecl fpclassify(float)" (?fpclassify@@YAHM@Z) 中引用了该符号
Detour.lib(DetourNavMeshQuery.cpp.obj) : error LNK2019: 无法解析的外部符号 _Init_thread_epoch，函数 "bool __cdecl dtVequal(float const *,float const *)" (?dtVequal@@YA_NPEBM0@Z) 中引用了该符号
Detour.lib(DetourNavMeshQuery.cpp.obj) : error LNK2019: 无法解析的外部符号 _tls_index，函数 "bool __cdecl dtVequal(float const *,float const *)" (?dtVequal@@YA_NPEBM0@Z) 中引用了该符号
Recast.lib(RecastMeshDetail.cpp.obj) : error LNK2001: 无法解析的外部符号 _tls_index
Recast.lib(RecastRegion.cpp.obj) : error LNK2019: 无法解析的外部符号 "void __cdecl `eh vector constructor iterator'(void *,unsigned __int64,unsigned __int64,void (__cdecl*)(void *),void (__cdecl*)(void *))" (??_L@YAXPEAX_K1P6AX0@Z2@Z)，函数 "bool __cdecl rcBuildRegions(class rcContext *,struct rcCompactHeightfield &,int,int,int)" (?rcBuildRegions@@YA_NPEAVrcContext@@AEAUrcCompactHeightfield@@HHH@Z) 中引用了该符号
Recast.lib(RecastRegion.cpp.obj) : error LNK2019: 无法解析的外部符号 "void __cdecl `eh vector destructor iterator'(void *,unsigned __int64,unsigned __int64,void (__cdecl*)(void *))" (??_M@YAXPEAX_K1P6AX0@Z@Z)，函数 "bool __cdecl rcBuildRegions(class rcContext *,struct rcCompactHeightfield &,int,int,int)" (?rcBuildRegions@@YA_NPEAVrcContext@@AEAUrcCompactHeightfield@@HHH@Z) 中引用了该符号
C:\Program Files (x86)\Windows Kits\10\\lib\10.0.22621.0\\um\x86\kernel32.lib : warning LNK4272:库计算机类型“x86”与目 标计算机类型“x64”冲突
C:\Program Files (x86)\Windows Kits\10\\lib\10.0.22621.0\\um\x86\user32.lib : warning LNK4272:库计算机类型“x86”与目标 计算机类型“x64”冲突
C:\Program Files (x86)\Windows Kits\10\\lib\10.0.22621.0\\um\x86\gdi32.lib : warning LNK4272:库计算机类型“x86”与目标计算机类型“x64”冲突
C:\Program Files (x86)\Windows Kits\10\\lib\10.0.22621.0\\um\x86\winspool.lib : warning LNK4272:库计算机类型“x86”与目 标计算机类型“x64”冲突
C:\Program Files (x86)\Windows Kits\10\\lib\10.0.22621.0\\um\x86\shell32.lib : warning LNK4272:库计算机类型“x86”与目标计算机类型“x64”冲突
C:\Program Files (x86)\Windows Kits\10\\lib\10.0.22621.0\\um\x86\ole32.lib : warning LNK4272:库计算机类型“x86”与目标计算机类型“x64”冲突
C:\Program Files (x86)\Windows Kits\10\\lib\10.0.22621.0\\um\x86\oleaut32.lib : warning LNK4272:库计算机类型“x86”与目 标计算机类型“x64”冲突
C:\Program Files (x86)\Windows Kits\10\\lib\10.0.22621.0\\um\x86\uuid.lib : warning LNK4272:库计算机类型“x86”与目标计 算机类型“x64”冲突
C:\Program Files (x86)\Windows Kits\10\\lib\10.0.22621.0\\um\x86\comdlg32.lib : warning LNK4272:库计算机类型“x86”与目 标计算机类型“x64”冲突
C:\Program Files (x86)\Windows Kits\10\\lib\10.0.22621.0\\um\x86\advapi32.lib : warning LNK4272:库计算机类型“x86”与目 标计算机类型“x64”冲突
C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.39.33519\lib\x86\MSVCRT.lib : warning LNK4272:库计算机类型“x86”与目标计算机类型“x64”冲突
C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.39.33519\lib\x86\msvcprt.lib : warning LNK4272:库计算机类型“x86”与目标计算机类型“x64”冲突
_deps\recastnavigation-build\RecastBuilder\RecastBuilder.exe : fatal error LNK1120: 59 个无法解析的外部命令
[9/168] Building C object CMakeFiles\sqlite3.dir\_deps\sqlite3-src\shell.c.obj
D:\fgm\carlaUE5\CarlaUE5\Build\_deps\sqlite3-src\shell.c(26338): warning C4566: 由通用字符名称“\uFEFF”表示的字符不能在当前代码页(936)中表示出来
[17/168] Building CXX object LibCarla\CMakeFiles\carla-server.dir\source\carla\streaming\detail\tcp\Server.cpp.obj^C
D:\fgm\carlaUE5\CarlaUE5>


does anyone have seen this ?

--- a/CMake/Common.cmake
+++ b/CMake/Common.cmake
@@ -39,6 +39,7 @@ endif ()
 
 if (WIN32)
   add_compile_definitions (_CRT_SECURE_NO_WARNINGS)
+  add_compile_options (/utf-8)
 endif ()
 
 set (CARLA_COMMON_DEFINITIONS)

--- a/CMake/Common.cmake
+++ b/CMake/Common.cmake
@@ -39,7 +39,11 @@ endif ()
 
 if (WIN32)
   add_compile_definitions (_CRT_SECURE_NO_WARNINGS)
-  add_compile_options (/utf-8)
+
+  check_cxx_compiler_flag(/utf-8 HAS_MSVC_UTF8)
+  if (HAS_MSVC_UTF8)
+    add_compile_options (/utf-8)
+  endif ()
 endif ()
 
 set (CARLA_COMMON_DEFINITIONS)

